sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/TriggerBindings/ServerlessHubOfT.cs

```diff
@@ -42,15 +43,29 @@ public abstract class ServerlessHub<T> where T : class
         /// <summary>
         /// Get the name of the hub.
         /// </summary>
-        public string HubName => typeof(T).Name;
+        public string HubName => GetType().Name;
```

Comment:
Why did we make this change? Won't this give us `ServerlessHub<Foo>` rather than `Foo`? Is that what we want?

Reply:
This is by design. If users have a `ChatHub : ServerlessHub<IChatClient>`, then the hub name is `ChatHub` other than `IChatClient`. This is consistent with `ServerlessHub` and the the `Hub` in SignalR.